crema/dataset.py

```diff
@@ -30,6 +30,10 @@ class PsmDataset:
         indicated either in square brackets :code:`[]` or parentheses
         :code:`()`. The exact modification format within these entities does
         not matter, so long as it is consistent.
+    peptide_pairing: dict
+        A map of target and decoy peptide sequence pairings to be used for TDC.
+        This should be in the form {key=target_sequence:value=decoy_sequence}
```

Comment:
I'm not sure. The way it works right now follows this logic:
- if a target has a corresponding decoy, then I pair target:decoy and decoy:decoy.
- if a target does not have a corresponding decoy, then I pair target:target.

This is because later when I compete, matching targets/decoys should have the same "pairing" field (which would be the decoy sequence by how I did it). Targets without matches are "paired" with their own sequence; this way they won't lose to a decoy sequence because no decoy sequence will be paired with a target sequence.

Reply:
I actually just meant the documentation. I'm fairly certain there is a standard way to document the keys and values of a dictionary 😄